easybuild/easyconfigs/r/reframe/reframe-3.0-dev2.eb

```diff
@@ -17,15 +20,28 @@ sources = [
                     '%(name)s/reframe-tests-%(version)s.tar.gz'
     }
 ]
-dependencies = [('Python-bare', '3.6.10')]
+
+dependencies = [('setuptools', '41.0.1', '-py3'),
+                ('pip', '20.0.2', '-py3')]
+builddependencies = [('pip', '20.0.2', '-py3')]
+
 keepsymlinks = True
 postinstallcmds = [
     'cp %(installdir)s/config/cscs.py %(installdir)s/reframe/settings.py',
     'cp -r %(builddir)s/reframe-tests-%(version)s/checks %(installdir)s/cscs-checks/private',
-    'mv %(installdir)s/cscs-checks %(installdir)s/checks'
+    'mv %(installdir)s/cscs-checks %(installdir)s/checks',
+    'PYTHONUSERBASE=%(installdir)s/installed_by_pip pip install --user -r %(builddir)s/reframe-%(version)s/requirements.txt',
```

Comment:
Isn't it better to have jsonschema as a depenency in `dependencies`?

Reply:
I install everything with pip via the `requirements.txt`.